src/NuGet.Core/NuGet.CommandLine.XPlat/ListPackage/ListPackageJsonRendererV1.cs

```diff
@@ -0,0 +1,384 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System;
+using System.IO;
+using Newtonsoft.Json;
+using System.Linq;
+using NuGet.Common;
+using NuGet.Protocol;
+using NuGet.Configuration;
+using NuGet.Versioning;
+
+namespace NuGet.CommandLine.XPlat.ListPackage
+{
+    /// <summary>
+    /// V1 of json format output renderer
+    /// </summary>
+    internal class ListPackageJsonRendererV1 : ListPackageJsonRenderer
```

Comment:
Is this an overoptimization? 

I'd argue a lot of this code would be something that could be shared among V1 and V2 of the potential rendering. 

Given that we don't have a V2 json right now, imo, we don't need this. 
We should just merge the 2 classes.

Reply:
Sure, considering all classes here are `internal` we can change as it fits next time, no problem with merging. Merged.